packages/expo-screen-capture/ios/EXScreenCapture/EXScreenCaptureModule.m

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXScreenCapture/EXScreenCaptureModule.h>
+
+@interface EXScreenCaptureModule ()
+
+@property (nonatomic, weak) UMModuleRegistry *moduleRegistry;
+@property (nonatomic, strong) UIView *blockView;
+
+@end
+
+@implementation EXScreenCaptureModule
+
+UM_EXPORT_MODULE(ExpoScreenCapture);
+
+- (id)init {
+  if (self = [super init]) {
+      self.blockView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, [[UIScreen mainScreen] bounds].size.height, [[UIScreen mainScreen] bounds].size.height)];
+      [self.blockView setBackgroundColor:[UIColor blackColor]];
+  }
+  return self;
+}
+
+- (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
+{
+  _moduleRegistry = moduleRegistry;
+}
+
+UM_EXPORT_METHOD_AS(activatePreventScreenCapture,
+                    activatePreventScreenCaptureWithResolver:(UMPromiseResolveBlock)resolve
+                    reject:(UMPromiseRejectBlock)reject)
+{
+  if (@available(iOS 11.0, *)) {
+    // If already recording, block it
+    dispatch_async(dispatch_get_main_queue(), ^{
+            [self preventScreenRecording];
+    });
+
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(preventScreenRecording) name:UIScreenCapturedDidChangeNotification object:nil];
+  }
+    
+  // This notification is only sent AFTER a screenshot is already taken. There is currently no exposed listner for iOS that sends an event BEFORE a screenshot.
+  // [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(screenShotWasTaken) name:UIApplicationUserDidTakeScreenshotNotification object:nil];
```

Comment:
Remove?

Reply:
Yeah, was keeping this around in case we decided to add a listener for this so that users _could_ take action if a user screenshotted their app (i.e. like snapchat) but suppose out of scope for now